pkg/cmd/server/server.go

```diff
@@ -213,7 +212,7 @@ func NewCommand(f client.Factory) *cobra.Command {
 	command.Flags().StringVar(&config.pluginDir, "plugin-dir", config.pluginDir, "Directory containing Velero plugins")
 	command.Flags().StringVar(&config.metricsAddress, "metrics-address", config.metricsAddress, "The address to expose prometheus metrics")
 	command.Flags().DurationVar(&config.backupSyncPeriod, "backup-sync-period", config.backupSyncPeriod, "How often to ensure all Velero backups in object storage exist as Backup API objects in the cluster. This is the default sync period if none is explicitly specified for a backup storage location.")
-	command.Flags().DurationVar(&config.podVolumeOperationTimeout, "restic-timeout", config.podVolumeOperationTimeout, "How long backups/restores of pod volumes should be allowed to run before timing out.")
+	command.Flags().DurationVar(&config.podVolumeOperationTimeout, "pod-volume-timeout", config.podVolumeOperationTimeout, "How long backups/restores of pod volumes should be allowed to run before timing out.")
```

Comment:
Doesn't this have the same problem that we are exposing the concept `pod-volume` to end user?

Reply:
There is an existing annotation, which is used for the same purpose, that is named with "pod-volume", so we just to keep it the same with the annotation:
PodVolumeOperationTimeoutAnnotation = "velero.io/pod-volume-timeout"

Otherwise, if we want to change this also, we need to change another user interface thing -- annotation, do we need to do this?

Reply:
As the discussion, we will change the flag only, because the annotation has not been in any of Velero document. The flag name will be `fs-backup-timeout`

Reply:
Done